releasenotes/notes/terminating-headless.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: traffic-management
+issue:
+  - 47348
+releaseNotes:
+  - |
+    **Fixed** an issue causing traffic to terminating headless service instances to not function correctly.
```

Comment:
can you give more info, it occurs after a full push or just a eds

Reply:
You cannot send to the terminating service, as its configuration is removed.